From 2c06cfe1f770c4adc3325f553f474a632d6c23ec Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Thu, 11 Oct 2018 13:47:03 +0800
Subject: [PATCH 4822/5242] MLK-19880 ARM64: dts: 8qm: remove jpegdec/env from
 domu

commit  3e847a19d414ed7907e763722cf74dc92c912828 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove JPEGDEC/ENC, they are not used by domu.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index 9cb25f23..9ee3cfc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -915,6 +915,8 @@
 /delete-node/ &isi_6;
 /delete-node/ &isi_7;
 /delete-node/ &mipi_csi_1;
+/delete-node/ &jpegdec;
+/delete-node/ &jpegenc;
 
 &gpio0_mipi_csi0 {
 	status = "okay";
-- 
1.7.9.5

